yarn/metadata.csv

```diff
@@ -22,7 +22,7 @@ yarn.metrics.lost_nodes,gauge,,node,,The number of lost nodes,0,yarn,nds lst,
 yarn.metrics.unhealthy_nodes,gauge,,node,,The number of unhealthy nodes,0,yarn,nds unhlthy,
 yarn.metrics.decommissioned_nodes,gauge,,node,,The number of decommissioned nodes,0,yarn,nds decomm,
 yarn.metrics.rebooted_nodes,gauge,,node,,The number of rebooted nodes,0,yarn,nds rbtd,
-yarn.apps.progress_gauge,gauge,,percent,,The progress of the application as a percent,0,yarn,app prog,
+yarn.apps.progress_gauge,gauge,,percent,,The progress of the application as a percent, displayed as 0, 10, & 100, which represent the 3 states: hasn't started, in progress, & completed,0,yarn,app prog,
```

Comment:
```suggestion
yarn.apps.progress_gauge,gauge,,percent,,"The progress of the application as a percent, displayed as 0, 10, & 100, which represent the 3 states: hasn't started, in progress, & completed",0,yarn,app prog,
```
Your text looks good, but the extra commas break the CSV file format. I did a Google search and found a StackOverflow answer that says that you can enclose a CSV entry in quotes, so perhaps try that? I'm not familiar with how our software systems parse CSVs.

Reply:
Ahh good eye, thank you. I've added the quotes and will check if the build still fails.